EventTicketingSystem.ConsoleTest/ConfigurationProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace EventTicketingSystem.ConsoleTest
+{
+    internal static class ConfigurationProvider
```

Comment:
SOLID violation: intetrface : implementation

Reply:
It is just a test project.